telemetry/server.go

```diff
@@ -349,7 +350,59 @@ func (s *Server) rateLimit(next http.Handler) http.Handler {
 	})
 }
 
+func (s *Server) createWakuTelemetry(w http.ResponseWriter, r *http.Request) {
+	start := time.Now()
+	var telemetryData []WakuTelemetryRequest
+	decoder := json.NewDecoder(r.Body)
+	if err := decoder.Decode(&telemetryData); err != nil {
+		log.Println(err)
+		http.Error(w, "Failed to decode telemetry data", http.StatusBadRequest)
+		return
+	}
+
+	var errorDetails []map[string]interface{}
```

Comment:
Wouldn't it be better to create a type instead?
```
type ErrorDetail struct {
  Id string `json:"Id"`
  Error string `json:"Error"`
}
```
And then use that type in 
```
var errorDetails []map[string]ErrorDetail
```

Reply:
added and included in http response when errors are encountered